src/components/button/button.scss

```diff
@@ -32,32 +32,34 @@ button {
         width: 100%;
 
         &:not(:disabled) {
-            @include mixins.is-elevated-clickable(
-                $color: var(--mdc-theme-primary, rgb(var(--color-teal-default)))
-            );
             @include mixins.visualize-keyboard-focus;
+            @include mixins.is-elevated-clickable();
```

Comment:
I've watched the video on the issue, but I don't understand what problem it's trying to show. When comparing this branch to `next`, I did notice that the problem with the text changing colour when hovering is gone in this PR, so that's nice, but apart from that, I really can't see what has changed!? 🤷‍♂️

Reply:
the difference is that MDC only animates `box-shadow`.
our mixins animate `box-shadow`, `background-color`, and `transform`. So the pressing-down animation looks better and smoother with ours.

Reply:
who will press rebase and merge? :D